srcpkgs/audit/template

```diff
@@ -1,11 +1,12 @@
 # Template file for 'audit'
 pkgname=audit
-version=2.8.4
-revision=2
+version=2.8.5
+revision=1
 build_style=gnu-configure
 configure_args="--libdir=/usr/lib --enable-shared=audit --enable-gssapi-krb5
  --with-apparmor --with-libcap-ng --with-python --with-python3"
-hostmakedepends="automake libtool pkg-config intltool python python3 swig"
+hostmakedepends="automake libtool pkg-config intltool python-devel python3-devel
```

Comment:
`python*-devel` is most likely not needed.

Reply:
Yeah, left over from a half assed attempt to fix ARM, then I forgot about it.